staramr/detection/AMRDetection.py

```diff
@@ -93,6 +96,46 @@ def _create_plasmidfinder_dataframe(self, plasmidfinder_blast_map: Dict[str, Bla
                                                                genes_to_exclude=self._genes_to_exclude)
         return plasmidfinder_parser.parse_results()
 
+    def _generate_empty_columns(self, row: list, max_cols: int, cur_cols: int) -> list:
+        if(cur_cols < max_cols):
+            for i in range(max_cols-cur_cols):
+                row.append('-')
+
+        return row
+
+    def _create_mlst_dataframe(self, mlst_data: str) -> DataFrame:
+
+        columns = ['Isolate ID', 'Organism', 'Sequence Type']
```

Comment:
Could you change `Organism` to `Scheme`? This is a more descriptive name for this type of data.

Reply:
Fixed this in commit ac0a220